index.js

```diff
@@ -1,38 +1,30 @@
 #!/usr/bin/env node
 
-const exect = require('child_process').exec;
+const { exec } = require('child_process');
+const { promisify } = require('util');
 const path = require('path');
 const fs = require('fs');
 
+const execPromise = promisify(exec);
+
 const mainPath = path.dirname(fs.realpathSync(__filename));
 const soundPath = path.join(mainPath, './oloquinho');
+const windowsScript = path.join(mainPath, './forWindows.vbs');
 
-const oloquinho = function (){
-    const linuxcmd = 'paplay '+soundPath+'.ogg';
-    const windowscmd = path.join(mainPath, './forWindows.vbs')+' '+soundPath+'.mp3';
-    const maccmd = 'afplay '+soundPath+'.mp3';
+const oloquinho = () => {
+    const commandsForEachPlatform = {
+      linux: `paplay ${soundPath}.ogg`,
+      windows: `${windowsScript} ${soundPath}.mp3`,
+      mac: `afplay ${soundPath}.mp3`,
```

Comment:
Faltou usar os valores corrects do platform `win32` e `darwin`

Reply:
boa, valeu! Ja to mundando.